data/korath/korath weapons.txt

```diff
@@ -577,13 +737,170 @@ effect "slicer impact"
 	"random velocity" .1
 	"velocity scale" -.002
 
+
+outfit "Digger Mining Turret"
+	category "Turrets"
+	cost 509000
+	thumbnail "outfit/korath digger turret"
+	"mass" 16
+	"outfit space" -16
+	"weapon capacity" -14
+	"asteroid scan power" 15
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/digger"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/digger turret"
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 1.5
+		"turret turn" 4.6
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 2.9
+		"firing heat" 4.9
+		"shield damage" 0.8
+		"hull damage" 1.3
+		"minable damage" 5.2
+		"disruption damage" .08
+	description `When the Korath Exiles needed a cheap mining laser for their large vessels, they re-engineered this ancient but robust design. The mild shield disruption effect makes it somewhat useful in combat if required.`
+
+outfit "Digger Mining Beam"
+	category "Guns"
+	cost 419000
+	thumbnail "outfit/korath digger"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -8
+	"asteroid scan power" 10
+	"gun ports" -1
+	weapon
+		sprite "projectile/digger thin"
+			"frame rate" 20
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 0.1
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy"  2.1
+		"firing heat" 4.3
+		"shield damage" 0.5
+		"hull damage" 0.9
+		"minable damage" 4.6
+		"disruption damage" .06
+	description `The gun version of the classic Korath mining laser allows a larger mothership to release a cloud of fighter-sized miners to harvest resources when working faster-moving or more dangerous asteroid fields.`
+
+effect "korath digger hit"
+	sprite "effect/korath digger hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 30
+	"velocity scale" .005
+
+
+outfit "Langrage Hyper-Heaver"
+	category "Secondary Weapons"
+	cost 3450000
+	thumbnail "outfit/korath heaver"
+	"mass" 46
+	"outfit space" -46
+	"weapon capacity" -46
+	"fuel capacity" 200
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/heaver hyperspace"
+			"frame rate" 12
+			rewind
+		"hardpoint sprite" "hardpoint/korath heaver"
+		icon "icon/korath heaver"
+		velocity 30
+		lifetime 8
+		"shield damage" -342
+		"hull damage" -342
+		"hit force" 3600
+		"hit effect" "heaver hyperspace hit"
+		sound "heaver"
+		submunition "Heaver Shard" 20
+		"turret turn" 2.2
+		"reload" 40
+		"burst reload" 12
+		"burst count" 6
+		"firing energy" 31
+		"firing heat" 190
+		"firing fuel" 12.5
+		"inaccuracy" 0.5
+	description `Langrage Hyper-Heavers raise the use of improvised ammunition to a fine art. A small Banisher propels scrap metal, nearby asteroids, compressed garbage, or other debris into an unstable hyperspace window.`
+	description `	At close range, the jumping langrage merely forces the enemy back. Further along, being forced to hyper transforms it into a rapidly-expanding cloud of deadly projectiles as it returns to normal space.`
```

Comment:
```suggestion
	description `Langrage Hyper-Heavers raise the use of improvised ammunition to a fine art. This turret converts scrap metal, nearby asteroids, compressed garbage, or other debris into a high speed debris field. The exact mechanism is unclear, but the internal workings bear some similarities to both Hyperdrives and Jump Drives.`
	description `	At close range, the jumping langrage merely forces the enemy back. Further along, being forced to hyper transforms it into a rapidly-expanding cloud of deadly projectiles as it returns to normal space.`
```

I like precise-sounding language, but a few things occurred to me while writing this:

1. Descriptions are from First Last's point of view, and it seems unlikely that our human textile weaver would understand much about how these worked even if they did find a Korath willing to explain them at depth. (less than a dozen humans understand how hyperdrives work, according to the Hyperdrive's description, after all. If a known tech that has been around for many centuries is still that poorly understood, I sincerely doubt that First is going to pick up details if if they have the best korath instructor in the galaxy trying to teach them.)
2. It seems unlikely that the Korath would provide a detailed explanation.
3. Exactly how hyperdrives, hyperspace, and related techs could be weaponized is a matter of discussion. In order to avoid having this detail bog us down forever, this removes the details of exactly how it works, and instead hints that the weapon bears some similarities to them. This hints at the lore many of us would like to keep with the Heaver, without providing concrete details that could be problematic later.
4. I personally find descriptions more impactful when they are telling me what I can see and/or notice about the outfit, rather than merely providing a technical description. This gives me impressions and ideas, and leaves me to work out and/or speculate for myself.

Reply:
I really don't like the idea of switching from Hyperdrives to Jump Drives.  How is it acceptable to have the weapon use something as advanced as a JD but *not* a HD?  Isn't this making the alleged problem even worse?

The Exiles can only barely make jump drives.  They might have learned to make them more safely since their last contact with the Efreti, but I certainly wouldn't expect to see them being applied to other areas just yet.  Rather, this is a realm of technology better suited to a T3 civilisation, one for whom JD tech is old hat, not an exciting, dangerous, and not-universally-implemented field.

Worse, jump drives do not provide realspace acceleration from any evidence portrayed thus far.  That's a lot more lore-breaking to me than contradicting some primitive human physicists.

I rather prefer something along the lines of the earlier description you provided on Discord
```
    description `Langrage Hyper-Heavers raise the use of improvised ammunition to a fine art. This turret converts scrap metal, nearby asteroids, compressed garbage, or other debris into a high speed debris field. The exact mechanism is unclear, but the internal workings bear some similarities to hyperdrives.`
    description `    At close range, the jumping langrage merely forces the enemy back. Further along, being forced to hyper transforms it into a rapidly-expanding cloud of deadly projectiles as it returns to normal space.`
```

Reply:
You are jumping to conclusions, which the description encourages, but it also deliberately does not confirm that conclusion. :) 

First off, it doesn't say that it is using a jump drive; just that it has some superficial similarities to one. Given that the *entirety* of Korath tech is 100% alien and *far* above the tech level of our pilot, that only really means that they're making use of some of the more advanced parts of their tech. It *may* be bits of JD tech in there, but it is just as likely as, say, me tearing open a sidewinder missile and seeing computer chips and going "Hey, they must be using a PC in here!" because that's what I'm familiar with. There might be some shared components, probably more than a few shared support technologies, but as to the key stuff that really makes it special.... Who knows?

Secondly, the Korath so far haven't really shown much attention to hyper*drive* technology. Personally, I don't think there is a linear progression from HD to JD, but rather many possible options. We do, however, know that they have JD, and a fairly solid mastery of it (not just humanity's "only a dozen people alive know how HD work!") because otherwise it wouldn't have necessitated shattering their civilization in two to exile those with the knowledge.

As such, even assuming that it *is* in fact JD based stuff in there, that's merely building on tech they already know and have a solid grasp of, rather than exploring a completely different line of technological development (creating micro hyperdrive lines) that they have hithertoo not shown any interest in. If we want them to get into that stuff too, great. But I don't think it is necessary. If we're wanting to limit them, technologically speaking, I think it's a bigger limitation to have them just using a variation on the JD they already know than to have them *also* figuring out creating hyperspace lines.

Anyway, that's my reasoning behind it. Basically focusing in on their known tech, rather than expanding their knowledge to include other areas. 

If you still want me to revert to the older description, I can happily do so.

Reply:
If you expect me to think JDs are not involved in that description, than you may expect HDs to not be involved in the most recently pushed description based on your previous edit.

If Jump Drive technology were solidly mastered, I don't expect you'd see so many vessels *not* using them.  True, we have that implemented for gameplay reasons -- to prevent their largest ships from leaving Korath space and/or getting captured by the player -- but in this case, the gameplay is informing the lore, and it's a sensible point.  *Quarg* have mastered Jump Drives, and the Korath, try as they might, are not on their level.

As for their ability to master the hyperdrive, on that we disagree.  Perhaps learning to build JDs may have leapfrogged them beyond creating stable links, but I'd expect them to be far beyond what humans can do in that area.  Nor does the current description even mention links or windows, it's using your vague language.

Reply:
Of note is that in MZ's lore dump, it's stated that hyperdrive and jump drive technologies are distinctly separate from one another, and there's nothing stopping a faction from developing jump drives before hyperdrives, suggesting that jump drive tech isn't simply an extension of hyperdrive tech and is instead a separate tree entirely, albeit probably more difficult to start developing than hyperdrive tech, hence why we see hyperdrive tech come first so often.

Reply:
I'm good with the modified version that has been implemented. I do like the idea that HD and JD are actually different tech trees.